### PR TITLE
Formatted File

### PR DESCRIPTION
**Pull request by laueian**
## Summary
This is just some formatting changes
- 1
- 2
- 3

### CODE REVIEW

File: array_1.py [hunk 1]
```diff
@@ -1,333 +1,339 @@
...
+    else:
+        for a_tag in soup.find_all("a"):
+            list_of_urls_on_page.append([a_tag.get("href"), a_tag.text])
 
     print("---------------------------- beginning of iteration")
     print(metadata)
-    print(count, metadata['title']," - ",metadata['url'])
-    count = count+1
+    print(count, metadata["title"], " - ", metadata["url"])
+    count = count + 1
```
Comment (annemarielaue) [inline array_1.py:335]: Why did you add a space here around the plus sign? I don't like that formatting.

Comment (laueian) [reply array_1.py:335]: this is not a democracy

**Comment by annemarielaue** [comment]
LGTM!

**Review by annemarielaue — APPROVED** [review]
LGTM!
